common/scaffolding/reports2/text-level-en.html.html

```diff
@@ -0,0 +1,151 @@
+
+<html>
+<body>
```

Comment:
To remove and replace it by a page front-matter.

```suggestion
---
{
	"title": "Text level working example - Scaffolding",
	"language": "en",
	"altLangPage": "text-level-fr.html",
	"dateModified": "2023-03-09"
}
---
```